src/app/date_arrangements.js

```diff
@@ -0,0 +1,38 @@
+export class DateArrangements {
+    static date = new Date()
+    static month = () => this.date.getMonth();
+    static year = () => this.date.getFullYear();
```

Comment:
Incorrect usage of static fields, date should be a member of the class if you are using it's reference in other methods. These methods should also  not be static.

If you want to have static class, use pure functions/methods.